src/Generator/Generators/CSharp/CSharpSources.cs

```diff
@@ -3067,7 +3066,7 @@ private string GetLibraryOf(Declaration declaration)
             string libName = declaration.TranslationUnit.Module.SharedLibraryName;
 
             NativeLibrary library;
-            Context.Symbols.FindLibraryBySymbol(((IMangledDecl) declaration).Mangled, out library);
+            Context.Symbols.FindLibraryBySymbol(((IMangledDecl)declaration).Mangled, out library);
```

Comment:
In general - restore the space after a type case, you removed it in many places. If necessary, change your formatting settings in Visual Studio.